packages/typespec-ts/test/unit/sample/generateSampleContent.spec.ts

```diff
@@ -0,0 +1,382 @@
+          booleanLiteral: true,
+          numberLiteral: 1,
+          offsetDate: "{Your offsetDate}",
+          nullable: null,
+          nullableString: "{Your nullableString}",
+          unknownValue: "Unknown Type"
+        };
+        assert.isNotNull(mockStr);
+        assert.deepEqual(JSON.parse(mockStr!), res);
+      });
+
+      it("date", async () => {
+        const schemaMap = await emitSchemasFromTypeSpec(
+          `
+        model Test {
+          date: utcDateTime;
+        }
+        op getModel(@body input: Test): void;
+        `
+        );
+        const mockStr = generateParameterTypeValue(
+          "Test",
+          "input",
+          buildSchemaObjectMap({ schemas: schemaMap } as RLCModel)
+        );
+        assert.isNotNull(mockStr);
+        assert.deepEqual(mockStr, `{"date": new Date()}`);
+      });
+
+      it("enum", async () => {
+        const schemaMap = await emitSchemasFromTypeSpec(
+          `
+        @fixed
+        enum FixedEnum {
+          English,
+          Chinese,
+        }
+
+        @doc("Translation Language Values")
+        enum ExtensibleEnum {
+          English,
+          Chinese,
+        }
+        model Test {
+          fixedEnum: FixedEnum;
+          extensibleEnum: ExtensibleEnum;
+          otherStringEnum: "Chinese";
+          otherLiteralUnion: "string1" | "string2";
+          otherString: string;
+          testExtensibleEnum: ExtensibleEnum;
+          testFixedEnum: FixedEnum;
+        }
+        op getModel(@body input: Test): void;
+        `,
+          true
+        );
+        const mockStr = generateParameterTypeValue(
+          "Test",
+          "input",
+          buildSchemaObjectMap({ schemas: schemaMap } as RLCModel)
+        );
+        assert.isNotNull(mockStr);
+        // console.log(mockStr);
+        const res = {
+          fixedEnum: "English",
+          extensibleEnum: "English",
+          otherStringEnum: "Chinese",
+          otherLiteralUnion: "string1",
+          otherString: "{Your otherString}",
+          testExtensibleEnum: "English",
+          testFixedEnum: "English"
+        };
+        assert.deepEqual(JSON.parse(mockStr!), res);
+      });
+
+      it("array", async () => {
+        const schemaMap = await emitSchemasFromTypeSpec(
+          `
+        model A {
+          foo: string;
+        }
+        model B {
+          bar: string;
+        }
+        model Test {
+          objectArray: A[];
+          complexArray: (A | B | string)[];
+          simpleArray: string[];
+          recordArray: Record<string>[];
+        }
+        op getModel(@body input: Test): void;
+        `,
+          true
+        );
+        const mockStr = generateParameterTypeValue(
+          "Test",
+          "input",
+          buildSchemaObjectMap({ schemas: schemaMap } as RLCModel)
+        );
+        assert.isNotNull(mockStr);
+        const res = {
+          simpleArray: ["{Your simpleArray}"],
```

Comment:
> Do we have different simple type hint for simple array ? like, "Your simple string array" here ?

It is not easy to do such thing because we recursively intercept the type info and the parent type would be lost in child one. I would prefer to continue collecting more this kind of requirement to see on how to improve the mock value not limited to string but also for other types.

Reply:
I am okay with that.